### ISSUE TITLE
Line break formatting doesn't work

### ISSUE THREAD

**Issue by tayyabferozi** [prettier/prettier-vscode#1386]
I am having this issue in prettier since the last time I re-installed VSCode and prettier. I am using the latest version of VSCode. I was expecting the extension to break a long line into multiple lines to make them easy to read.

**INPUT:**

![prettier](https://user-images.githubusercontent.com/57319255/82299374-49164200-996a-11ea-9cf0-9a836d31c21f.PNG)

**OUTPUT:**

![prettier2](https://user-images.githubusercontent.com/57319255/82299394-50d5e680-996a-11ea-8088-77c4bea078c3.PNG)

**EXPECTED:**

![prettier3](https://user-images.githubusercontent.com/57319255/82299566-867acf80-996a-11ea-81cb-5cfd566d3458.PNG)

I was watching a tutorial, the instructor uses the same extension, this works for him but not for me.

### Additional information

**VS Code Version**: 1.44.2(x64)

**Prettier Extension Version:**

Prettier Code Formatter / Prettier v3.14.1
Prettier Code Formatter / Esben Peterson v4.7.0
(I have tried both, but both didn't work)

**OS:** Windows 10 Pro

## Prettier Log Output

```
["INFO" - 12:12:31 am] Extension Name: esbenp.prettier-vscode.
["INFO" - 12:12:31 am] Extension Version: 4.7.0.
["INFO" - 12:12:31 am] Enabling prettier for languages
[
  "css",
  "graphql",
  "html",
  "javascript",
  "javascriptreact",
  "json",
  "json5",
  "jsonc",
  "less",
  "markdown",
  "mdx",
  "mongo",
  "postcss",
  "scss",
  "typescript",
  "typescriptreact",
  "vue",
  "yaml"
]
["INFO" - 12:12:31 am] Enabling prettier for range supported languages
[
  "graphql",
  "javascript",
  "javascriptreact",
  "json",
  "typescript",
  "typescriptreact"
]
["INFO" - 12:18:14 am] Enabling prettier for languages
[
  "css",
  "graphql",
  "html",
  "javascript",
  "javascriptreact",
  "json",
  "json5",
  "jsonc",
  "less",
  "markdown",
  "mdx",
  "mongo",
  "postcss",
  "scss",
  "typescript",
  "typescriptreact",
  "vue",
  "yaml"
]
["INFO" - 12:18:14 am] Enabling prettier for range supported languages
[
  "graphql",
  "javascript",
  "javascriptreact",
  "json",
  "typescript",
  "typescriptreact"
]
["INFO" - 12:18:34 am] Enabling prettier for languages
[
  "css",
  "graphql",
  "html",
  "javascript",
  "javascriptreact",
  "json",
  "json5",
  "jsonc",
  "less",
  "markdown",
  "mdx",
  "mongo",
  "postcss",
  "scss",
  "typescript",
  "typescriptreact",
  "vue",
  "yaml"
]
["INFO" - 12:18:34 am] Enabling prettier for range supported languages
[
  "graphql",
  "javascript",
  "javascriptreact",
  "json",
  "typescript",
  "typescriptreact"
]
["INFO" - 12:18:37 am] Enabling prettier for languages
[
  "css",
  "graphql",
  "html",
  "javascript",
  "javascriptreact",
  "json",
  "json5",
  "jsonc",
  "less",
  "markdown",
  "mdx",
  "mongo",
  "postcss",
  "scss",
  "typescript",
  "typescriptreact",
  "vue",
  "yaml"
]
["INFO" - 12:18:37 am] Enabling prettier for range supported languages
[
  "graphql",
  "javascript",
  "javascriptreact",
  "json",
  "typescript",
  "typescriptreact"
]
["INFO" - 12:18:48 am] Enabling prettier for languages
[
  "css",
  "graphql",
  "html",
  "javascript",
  "javascriptreact",
  "json",
  "json5",
  "jsonc",
  "less",
  "markdown",
  "mdx",
  "mongo",
  "postcss",
  "scss",
  "typescript",
  "typescriptreact",
  "vue",
  "yaml"
]
["INFO" - 12:18:48 am] Enabling prettier for range supported languages
[
  "graphql",
  "javascript",
  "javascriptreact",
  "json",
  "typescript",
  "typescriptreact"
]
["INFO" - 12:19:00 am] Enabling prettier for languages
[
  "css",
  "graphql",
  "html",
  "javascript",
  "javascriptreact",
  "json",
  "json5",
  "jsonc",
  "less",
  "markdown",
  "mdx",
  "mongo",
  "postcss",
  "scss",
  "typescript",
  "typescriptreact",
  "vue",
  "yaml"
]
["INFO" - 12:19:00 am] Enabling prettier for range supported languages
[
  "graphql",
  "javascript",
  "javascriptreact",
  "json",
  "typescript",
  "typescriptreact"
]
["INFO" - 12:19:03 am] Enabling prettier for languages
[
  "css",
  "graphql",
  "html",
  "javascript",
  "javascriptreact",
  "json",
  "json5",
  "jsonc",
  "less",
  "markdown",
  "mdx",
  "mongo",
  "postcss",
  "scss",
  "typescript",
  "typescriptreact",
  "vue",
  "yaml"
]
["INFO" - 12:19:03 am] Enabling prettier for range supported languages
[
  "graphql",
  "javascript",
  "javascriptreact",
  "json",
  "typescript",
  "typescriptreact"
]
["INFO" - 12:19:19 am] Enabling prettier for languages
[
  "css",
  "graphql",
  "html",
  "javascript",
  "javascriptreact",
  "json",
  "json5",
  "jsonc",
  "less",
  "markdown",
  "mdx",
  "mongo",
  "postcss",
  "scss",
  "typescript",
  "typescriptreact",
  "vue",
  "yaml"
]
["INFO" - 12:19:19 am] Enabling prettier for range supported languages
[
  "graphql",
  "javascript",
  "javascriptreact",
  "json",
  "typescript",
  "typescriptreact"
]
["INFO" - 12:19:30 am] Enabling prettier for languages
[
  "css",
  "graphql",
  "html",
  "javascript",
  "javascriptreact",
  "json",
  "json5",
  "jsonc",
  "less",
  "markdown",
  "mdx",
  "mongo",
  "postcss",
  "scss",
  "typescript",
  "typescriptreact",
  "vue",
  "yaml"
]
["INFO" - 12:19:30 am] Enabling prettier for range supported languages
[
  "graphql",
  "javascript",
  "javascriptreact",
  "json",
  "typescript",
  "typescriptreact"
]

```

### THANKS IN ADVANCE

**Comment by JHilker** [comment]
Based on your log it doesn't look like Prettier ever attempted to run on this file. Have you configured [VSCode to format on save](https://github.com/prettier/prettier-vscode#format-on-save)? Otherwise it will only format a file when you [explicitly trigger formatting](https://code.visualstudio.com/docs/editor/codebasics#_formatting).

**Comment by tayyabferozi** [comment]
> 
> 
> Based on your log it doesn't look like Prettier ever attempted to run on this file. Have you configured [VSCode to format on save](https://github.com/prettier/prettier-vscode#format-on-save)? Otherwise it will only format a file when you [explicitly trigger formatting](https://code.visualstudio.com/docs/editor/codebasics#_formatting).

I am using Windows and I tried to format by pressing `Shift + Alt + F`. Also I have tried to format by pressing `Ctrl + F` after selecting the whole script but still it didn't work. 

I have my editor's Format On Save also enabled.
![fos](https://user-images.githubusercontent.com/57319255/82754432-b1f22580-9d81-11ea-8022-635b057c66bb.png)

I don't know why it doesn't show in the log. **Here is my** updated **log** (I am using this on another script this time).
`["INFO" - 5:32:01 am] Extension Name: esbenp.prettier-vscode.
["INFO" - 5:32:01 am] Extension Version: 4.7.0.
["INFO" - 5:32:01 am] Enabling prettier for languages
[
  "css",
  "graphql",
  "html",
  "javascript",
  "javascriptreact",
  "json",
  "json5",
  "jsonc",
  "less",
  "markdown",
  "mdx",
  "mongo",
  "postcss",
  "scss",
  "typescript",
  "typescriptreact",
  "vue",
  "yaml"
]
["INFO" - 5:32:01 am] Enabling prettier for range supported languages
[
  "graphql",
  "javascript",
  "javascriptreact",
  "json",
  "typescript",
  "typescriptreact"
]
`
